crates/standalone/src/subcommands/start.rs

```diff
@@ -174,6 +180,12 @@ pub async fn exec(args: &ArgMatches) -> anyhow::Result<()> {
     } else {
         Storage::Disk
     };
+    let fsync = if args.get_flag("wal_fsync") {
```

Comment:
Ooh I think we can maybe make these Storage and FsyncPolicy enums `#[derive(clap::ValueEnum)]` and not do this manually, for later tho.